### PR TITLE
Add FileType parameter to Set-DbaDbFileGrowth

### PR DESCRIPTION
**Pull request by gbargsley**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [X ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/dataplat/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### Purpose
This allows you to change Data and Log growth settings independently.

### Approach

Added some logic and parameter to let you choose data or log files
### Commands to test

Example added

### CODE REVIEW

File: functions/Set-DbaDbFileGrowth.ps1 [hunk 1]
```diff
@@ -72,6 +75,12 @@ function Set-DbaDbFileGrowth {
         PS C:\> Set-DbaDbFileGrowth -SqlInstance sql2017, sql2016, sql2012 -Database test -WhatIf
 
         Shows what would happen if the command were executed
+
+    .EXAMPLE
+        PS C:\> Set-DbaDbFileGrowth -SqlInstance sql2017 -Database test -GrowthType GB -Growth 1 -FileType Data
+
+        Allows you to set growth to Data or Log independently
```
Comment (andreasjordan) [inline functions/Set-DbaDbFileGrowth.ps1:82]: I would say "Sets a filegrowth of 1 GB for the data files only". So tell the user what the example code does and not what the used parameter can do.

File: functions/Set-DbaDbFileGrowth.ps1 [hunk 2]
```diff
@@ -83,7 +92,9 @@ function Set-DbaDbFileGrowth {
         [int]$Growth = 64,
         [parameter(ValueFromPipeline)]
         [Microsoft.SqlServer.Management.Smo.Database[]]$InputObject,
-        [switch]$EnableException
+        [switch]$EnableException,
+        [ValidateSet('All', 'Data', 'Log')]
+        [string]$FileType = "All"
```
Comment (andreasjordan) [inline functions/Set-DbaDbFileGrowth.ps1:97]: Should be moved up between $Growth and $InputObject.

**Review by andreasjordan — CHANGES_REQUESTED** [review]
Thanks for the pull request.

File: functions/Set-DbaDbFileGrowth.ps1 [hunk 3]
```diff
@@ -29,6 +29,9 @@ function Set-DbaDbFileGrowth {
     .PARAMETER Growth
         The growth value. 64 by default.
 
+    .PARAMETER FileType
+        Apply changes to only DATA, LOG or ALL data files
```
Comment (andreasjordan) [inline functions/Set-DbaDbFileGrowth.ps1:33]: Maybe remove the "data"? "Apply changes to only DATA, LOG or ALL files"

**Review by andreasjordan — CHANGES_REQUESTED** [review]
You will have to also change the test file and add the new parameter there.

File: functions/Set-DbaDbFileGrowth.ps1 (hunk 3 above)
Comment (gbargsley) [reply functions/Set-DbaDbFileGrowth.ps1:33]: Made the data change.  I do not have my configuration setup to test the TESTS, so not sure what to add.

**Review by andreasjordan — APPROVED** [review]

**Comment by potatoqualitee** [comment]
Thank you both! 🙇🏼

**Pull request merged**